tokio/src/net/tcp/socket.rs

```diff
@@ -404,6 +404,51 @@ impl TcpSocket {
         self.inner.linger()
     }
 
+    /// Sets the value of the TCP_NODELAY option on this socket.
```

Comment:
Please be consistent about whether TCP_NODELAY is in backticks.

Reply:
Done!